application.py

```diff
@@ -1,10 +1,9 @@
 from flask import Flask
-ai_api = Flask(__name__)
+application = app = Flask(__name__)
```

Comment:
Why `application=app=Flask(__name__)?`

isn't `application=Flask(__name__)` enough?

Reply:
This is an odd issue with AWS' elastic beanstalk. It appears there are some conventions we need to be aware of in terms of application naming. We can talk this over next time we meet up. I'm not really happy with this as we should be able to name the app variable whatever we want. We should take a look in the future at refactoring away from elastic beanstalk.